src/Microsoft.DotNet.GenAPI/Program.cs

```diff
@@ -48,6 +48,7 @@ private static int Main(string[] args)
             CommandOption hightlightInterfaceMembers = app.Option("--hightlight-interface-members", "[CSDecl] Highlight interface implementation members.", CommandOptionType.NoValue);
             CommandOption alwaysIncludeBase = app.Option("--always-include-base", "[CSDecl] Include base types, interfaces, and attributes, even when those types are filtered.", CommandOptionType.NoValue);
             CommandOption excludeMembers = app.Option("--exclude-members", "Exclude members when return value or parameter types are excluded.", CommandOptionType.NoValue);
+            CommandOption langVersion = app.Option("--langVersion", "Language Version to target", CommandOptionType.SingleValue);
```

Comment:
```suggestion
            CommandOption langVersion = app.Option("--lang-version", "Language Version to target", CommandOptionType.SingleValue);
```

I suggest we keep consistent casing on option flags. The other options on this tool use kebab-case.

Reply:
Fixed.